packages/orbit-components/src/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import { select, text, boolean, number } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+
+import RenderInRtl from "../utils/rtl/RenderInRtl";
+import { TYPE_OPTIONS } from "./components/Tab/consts";
+
+import Tabs, { TabList, TabPanel, Tab, TabPanels } from ".";
+
+export const Default = () => {
+  return (
+    <Tabs onChange={action("onChange")}>
+      <TabList>
+        <Tab>Tab 1</Tab>
+        <Tab type="basic">Tab 2</Tab>
+        <Tab type="medium">Tab 3</Tab>
+        <Tab type="top">Tab 4</Tab>
+      </TabList>
+      <TabPanels>
+        <TabPanel>Tab 1 content</TabPanel>
+        <TabPanel>Tab 2 content</TabPanel>
+        <TabPanel>Tab 3 content</TabPanel>
+        <TabPanel>Tab 4 content</TabPanel>
+      </TabPanels>
+    </Tabs>
+  );
+};
+
+export const Controlled = () => {
+  const [selected, setSelected] = React.useState(0);
```

Comment:
Do this an you will see the bug.

```suggestion
  const [selected, setSelected] = React.useState(2);
```

Reply:
It's not a bug. defaultSelected is not supposed to be used with controlled. README md is mentioning it.